src/FlexVotingClient.sol

```diff
@@ -0,0 +1,209 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.10;
+
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {Checkpoints} from "@openzeppelin/contracts/utils/Checkpoints.sol";
+import {IFractionalGovernor} from "src/interfaces/IFractionalGovernor.sol";
+import {IVotingToken} from "src/interfaces/IVotingToken.sol";
+
+/// @notice This is an abstract contract to make it easy to build clients for
```

Comment:
Awesome explanation. Absolutely love this abstraction for third party devs 😍

Reply:
Thank you! 🙂